Gymphony/src/app/features/courses/services/course-mapper.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+
+import { DraftCourse } from '../interfaces/draft-course';
+import { Course } from '../interfaces/course';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CourseMapperService {
```

Comment:
I guess there is no need to create a service for mapping
It can be arrow function declared as const